### PR TITLE
Fix the endpoints

### PR DESCRIPTION
**Pull request by neha-bhargava**
Fixes # 
Fix the bug in dSTS

**Changes proposed in this request**
Fix the endpoints

**Testing**
Updated the unit test

**Performance impact**

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]
looks good `oauth2/v2.0` was requested by DSTS

**Review by pmaytak — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Unit/PublicApiTests/DstsTests.cs [hunk 1]
```diff
@@ -44,20 +44,18 @@ private static MockHttpMessageHandler CreateTokenResponseHttpHandler(string auth
...
             };
         }
 
         [DataTestMethod]
-        [DataRow("common")]
-        [DataRow("tenantid")]
-        public async Task DstsClientCredentialSuccessfulTestAsync(string tenantId)
+        [DataRow(CommonAuthority)]
+        [DataRow(TenantedAuthority)]
+        public async Task DstsClientCredentialSuccessfulTestAsync(string authority)
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Unit/PublicApiTests/DstsTests.cs:57]: nit: I recommend putting all dsts tests in 1 file.

**Review by bgavrilMS — APPROVED** [review]
Ok, but HTTP test should catch this sort of changes.

**Review by SameerK-MSFT — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/DstsAuthorityTests.cs [hunk 2]
```diff
@@ -1,58 +1,73 @@
...
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Instance;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Unit.CoreTests.InstanceTests
+{
+    [TestClass]
```
[the remark this answers is not in the document]
Comment (neha-bhargava) [reply tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/DstsAuthorityTests.cs:17]: The test was also changed with the endpoint. I fixed the test too

**Pull request merged**